Dockerfile

```diff
@@ -31,7 +31,15 @@ RUN \
     install_packages \
         wget \
         i2c-tools \
-        libdbus-1-3
+        libdbus-1-3 \
+        gpg
+
+# Nebra uses /opt by convention
+WORKDIR /opt/
+
+# Import gpg key
+COPY keys/manufacturing-key.gpg ./
+RUN gpg --import manufacturing-key.gpg
```

Comment:
@posterzh - once imported should we remove the key from /opt? Although keeping it there also does not affect anything.